src/CallableHandler.php

```diff
@@ -122,4 +75,13 @@ public static function execute($callable, array $arguments = [])
             throw $exception;
         }
     }
+
+    protected static function getResolver()
```

Comment:
Do not use static variables for this. Note that this is a helper that can be used by many middlewares with different containers